data_structures/queue_using_one_stack.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+  @author [Rajat Garg](https://github.com/Rajat379)
+  @file
+
+  Implementation of a Queue using one Stack.
+*/
+
+#include <cassert>
+#include <iostream>
+#include <stack>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```